src/test/java/embl/ebi/variation/commons/models/metadata/database/StudyDatabaseTest.java

```diff
@@ -40,6 +42,9 @@
     StudyRepository repository;
 
     @Autowired
+    AnalysisRepository analysisRepository;
```

Comment:
Added FileGeneratorRepository, testing both repositories in all tests that use AnalysisRepository